cdk/lib/website-stack.ts

```diff
@@ -5,11 +5,41 @@ import cdk = require("@aws-cdk/core");
 import s3 = require("@aws-cdk/aws-s3");
 import cf = require("@aws-cdk/aws-cloudfront");
 import iam = require("@aws-cdk/aws-iam");
+import cognito = require("@aws-cdk/aws-cognito");
 
 export class WebsiteStack extends cdk.Stack {
-  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+  constructor(
+    scope: cdk.Construct,
+    id: string,
+    props: cdk.StackProps,
+    config: any // FIXME Nail down typing here
```

Comment:
I'm not happy about this, but I want to unblock your work with Cognito.

https://github.com/aws-samples/busy-engineers-document-bucket/issues/9